mobile/src/controllers/LocalSettings.ts

```diff
@@ -113,9 +112,24 @@ export class LocalSettingsController implements ILocalSettingsController {
         await this._synced.triggerAsync();
     }
 
+    private submitChangesHealth = async () => {
+        const diff: Partial<UserLocalSettings> = {
+            health: toJS(this._current.health),
+        };
+
+        logger.log('[LocalSettingsController] submitting changes...', diff);
```

Comment:
Is this left here by mistake? If so, it should be removed (otherwise let me know why it's left here)